apps/edxapp/vars/all/main.yml

```
@@ -45,12 +45,14 @@ edxapp_mongodb_image_name: "centos/mongodb-32-centos7"
 edxapp_mongodb_image_tag: "3.2"
 edxapp_mongodb_port: 27017
 edxapp_mongodb_secret_name: "edxapp-mongo-{{ edxapp_vault_checksum }}"
+edxapp_mongodb_version: "3.2"
```

As the docker image tag matches required version, why not defining it only once, _e.g_:

```suggestion
edxapp_mongodb_version: "3.2"
edxapp_mongodb_image_name: "centos/mongodb-32-centos7"
edxapp_mongodb_image_tag: "{{ edxapp_mongodb_version }}"
```

---

This doesn't work